package.json

```diff
@@ -5,8 +5,10 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon src/index.ts"
+    "docker:migrate": "docker-compose exec service npx drizzle-kit push",
```

Comment:
`process.env.DATABASE_URL` isn't defined outside the Container, and I don't really love having to run it at all right now, so for now just make sure it runs within the Docker Container.

Will eventually add `--force` so it doesn't pause to ask.